flytekit/types/file/file.py

```diff
@@ -324,6 +325,60 @@ def assert_type(
     def get_literal_type(self, t: typing.Union[typing.Type[FlyteFile], os.PathLike]) -> LiteralType:
         return LiteralType(blob=self._blob_type(format=FlyteFilePathTransformer.get_format(t)))
 
+    def get_mime_type_from_python_type(self, extension: str) -> str:
```

Comment:
LGTM, one nit: could we rename it to `get_mime_type_from_extension`?

Reply:
updated

Reply:
https://github.com/flyteorg/flytekit/blob/ec732bda3083e8ae23fd42476fa6c1be4f87ebff/flytekit/types/file/file.py#L328